generic/generic_cmd-1.2.3.recipe

```diff
@@ -35,15 +44,25 @@ BUILD_PREREQUIRES="
 	cmd:gcc
 	"
 
+GLOBAL_WRITABLE_FILES="
+	settings/projectx.conf keep-old
+	"
+
+defineDebugInfoPackage projectx \
+	"$commandBinDir"/projectx
+
 BUILD()
 {
+	runConfigure --omit-dirs "binDir sbinDir" ./configure \
+		--bindir="$commandBinDir" \
+		--sbindir="$commandBinDir"
 	make $jobArgs
 }
 
 INSTALL()
 {
-	mkdir -p $binDir
-	cp -a objects/projectx $binDir
+	mkdir -p $commandBinDir
+	cp -a objects/projectx $commandBinDir
```

Comment:
I think we want to use `install` not `cp`, but @fbrosson will know better than I do on that.

Reply:
Yep, using `install -d` (instead of `mkdir -p`) and `install -t` (instead of `cp`) is nice because we could need, someday, to customise `install` to do a few extra things like setting the ownership, or choosing sane default modes.
That said, I don't think `cp -r` can be replaced with an `install`. But we don't have many `cp -r` I think.

But it is still a good idea to recommend `install`.

Regarding the modes, it seems we need to specify them when we install non executable files (i.e. docs), but I'd need to check this.
I think @alaviss knows this topic quite well.